scripts/generate

```diff
@@ -162,9 +139,10 @@ def('toOutputMarkup',
     {},
     [$.String, $.String],
     pipe([encaseEither(prop('message'),
-                       curry(vm.runInNewContext)(_,
-                                                 {R: R, S: S_, sqrt: sqrt},
-                                                 {displayErrors: false})),
+                       curry3(vm.runInNewContext,
+                              _,
+                              {R: R, S: S_, sqrt: sqrt},
+                              {displayErrors: false})),
```

Comment:
I :heart: the fact that `S.curry3` and friends support both left partial application and right partial application (thanks to `S.__`).